apicli/version.py

```diff
@@ -0,0 +1,5 @@
+from distutils.version import StrictVersion
+
+__all__ = ['VERSION']
+
+VERSION = StrictVersion('0.0.1')
```

Comment:
See https://github.com/pallets/flask/blob/master/setup.py#L52 almost every Python package do it its way